### PR TITLE
CORTX-28870: Adding the allowable list for Alex scan

### PR DESCRIPTION
**Pull request by venkuppu-chn**
Signed-off-by: Venkatesh K <venkatesh.k@seagate.com>

### CODE REVIEW

**Review by sachinpunadikar — APPROVED** [review]
+1

File: .alexrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "allow": ["execute", "host-hostess", "remains", "reject", "traps", "invalid", "period", "invalid", "execution", "simple", "failure"]
```
Comment (rahul-tripathi-git) [inline .alexrc:2]: invalid is twice in this list, please fix this.

**Review by rahul-tripathi-git — CHANGES_REQUESTED** [review]

**Review by mukul-seagate11 — APPROVED** [review]
looks good

File: .alexrc (hunk 1 above)
Comment (venkuppu-chn) [reply .alexrc]: Done

**Pull request merged**